Essentials/src/config.yml

```diff
@@ -563,6 +563,11 @@ last-message-reply-recipient: true
 # Default is 180 (3 minutes)
 last-message-reply-recipient-timeout: 180
 
+# Changes the default /reply functionality.
+# If true, /reply will not check if the person you're replying to has vanished.
+# If false, players will not be able to /reply to players who they can no longer see due to vanish.
+last-message-reply-vanished: true
```

Comment:
I believe it should be `false` by default
While it *is* changing behavior that some may not like, I think that it would be better that you won't be able to reply to vanished players (because of the whole vanished concept)

Reply:
up to @md678685 but I'm pretty sure keeping current behavior is the goal

Reply:
IMO this should be `false` for new configs but `true` for existing ones (default in `Settings`) - that way, new installs will have the new behaviour without changing behaviour on current servers.